ui/revenuecatui/build.gradle

```diff
@@ -76,6 +82,8 @@ dependencies {
     testImplementation libs.bundles.test
     testImplementation libs.coroutines.test
     testImplementation libs.kotlinx.serialization.json
+    testImplementation libs.androidx.test.compose
+    testImplementation libs.hamcrest.core
```

Comment:
Got all kinds of "no class definition" errors without this. Not entirely sure why it isn't transitively provided though.

Reply:
hmm, that's interesting! Since this is for testing, I would not super concerned though